fearless/Modules/Staking/StakingMain/StakingMainViewController.swift

```diff
@@ -343,6 +343,10 @@ extension StakingMainViewController: NetworkInfoViewDelegate {
     func animateAlongsideWithInfo(view _: NetworkInfoView) {
         scrollView.layoutIfNeeded()
     }
+
+    func viewCollapsed(_ value: Bool, view _: NetworkInfoView) {
+        presenter.networkInfoViewDidCollapsed(value)
```

Comment:
Why do we need view parameter here?

Reply:
jus for future needs, it's common for delegate to pass object. You know 'func tableView(_ tableView: UITableView, did...)'